README.md

```diff
@@ -103,7 +104,8 @@ curl http://127.0.0.1:3000/api/users/big -i -X PATCH \
   -F password=newpassword
 ```
 
-
+When using above calls on the live demo version, the user/password specified by `-u` should be the main `fullfact` account (password in Bitwarden); and the URL should be `https://raphael.fullfact.org/api/` etc. The account being modified (e.g. `big` in the above examples) will be part of the URL for the disable/change password calls.
```

Comment:
I think just changing the username in the examples from `big` to `alice` or `bob` might obviate the need for the final sentence.